benchmarks/src/main/scala/io/finch/benchmarks.scala

```diff
@@ -190,11 +197,13 @@ abstract class BootstrapBenchmark[CT](init: Bootstrap[Id, HNil, HNil])(implicit
 
   protected def issueRequest(): Request = Request()
 
-  private val foo: Service[Request, Response] =
+  private val foo: Resource[IO, Service[Request, Response]] =
     init.serve[CT](Endpoint[IO].const(List.fill(128)(Foo(scala.util.Random.alphanumeric.take(10).mkString)))).toService
 
   @Benchmark
-  def foos: Response = Await.result(foo.apply(issueRequest()))
+  def foos: Response = Await.result(dispatcher.unsafeRunSync(foo.use { service =>
```

Comment:
I think it's ok if we keep this as is for this change, but we'd need to revisit this sometime later. Ideally, we'd need to `use` the resource on each benchmark invocation. Maybe we could leverage before/after setup machine to manage the lifecycle of the resource.